application/authseeder.py

```diff
@@ -34,15 +34,25 @@ class AuthSeeder():
         "3":"Astronaut"  
     }
 
-    permissions = {"1":"create-user",        
-        "2":"aproove-user",        
-        "3":"delete-user",
-        "4":"assign-role",
-        "5":"update-record"  
+    permissions = {"create-user",        
```

Comment:
as this is a dictionary, each item should be a key-value pair. Theres probably a red squiggly line under this, or if you run one of the code stylers it should catch this. That being said, I'm not sure we need to delete the permission ids as they are used in the role_has_permissions variable below

Reply:
Hi Brad, no squiggly line and I thought that as well, but after our call I understood we were getting rid of role ids altogether. I should have changed it to a list and forgot to change the brackets.